docs/user/multi-user-dfuse.md

```diff
@@ -0,0 +1,111 @@
+
+## Unified Namespace.
+
+### Requirement for setting of ACLs.
+
+To make use of Unified Namespace with a single dfuse instance across processes for multiple users
+appropriate ACLs need to be set on both DAOS pools and containers.  DFuse is like any other daos
+client in this regard and when running multi-user dfuse then it is likely that the user dfuse is
+running as does not own all the pools and containers it will be serving.
+
+ACLs required for pools is the 'r' permission on any pool for the user running dfuse.  This
+administrator may choose to set this when creating pools.
+
+It is possible for a user to run multi-user dfuse and to do so purely to expose data in their own
+containers to other users. In this case the user running dfuse is the same as the user owning the
+containers so no modification of ACLs is required.
+
+### Automatic setting of ACLs.
+
+For containers the 'r', 't' and 'w' permissions are required on the container for the user running
+dfuse.  When containers are created though the `daos` command using a path which resides in a
+multi-user dfuse mount then these ACLs are set automatically and this is reported to the user.
+(TODO: Check this is reported).
+
+If a container is created using a path not backed by multi-user dfuse - for example, on a per-user
+dfuse instance or where a `path` option is not provided at all - then in order for anyone to access
+the container the user will need to add this permission themselves.  This includes the user that
+owns the pool. Without the dfuse users being able to access the container it cannot serve the
+contents, even to the container owner.  This applies equally well to containers where the Unified
+Namespace link is created after the container itself is created.
+
+
+## Controlling data sharing and access.
+
+DFuse itself does not handle permissions checks regardless of options but rather provides file and
+directory ownership and permissions bits information to the kernel which then checks if user is
+entitled to perform the requested operation.  As such once a multi-user dfuse instance is running it
+will serve to the kernel the contents of any containers that the dfuse instance itself can access,
+and the kernel will then use standard Unix file/directory permissions to decide if operations are
+permitted.  It is therefore possible for users be able to access POSIX data across containers
+seamlessly where permissions is granted only at the POSIX file/directory layer.
+
+Consider the following example: dfuse is configured to run in the recommended manner, with two users
+Anthony and Berlinda who use it to share data, yet Berlinda does not have ACL permissions to read
+Anthony's data - only POSIX permissions.
+
+Example:
+
+### Setup user to serve dfuse and create containers, mount points etc.
+```bash
+$ sudo -u dserve dmg --insecure pool create root_pool --size 1g
+$ sudo -u dserve daos cont create --type POSIX root_pool root_container
+$ sudo mkdir /crate
```

Comment:
rename /crate to /daos ?

Reply:
We've previously had problems with users getting confused between and server mounts/fileystems etc and the dfuse one which if why I'm shying away from calling a dfuse mount "/daos" but I can change it if you think it's better.